source/Calamari/Calamari.csproj

```diff
@@ -16,7 +16,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Calamari.Common" Version="19.0.1-enh-logwithconte0003" />
```

Comment:
This version is being downgraded because it should not have been using `Calamari.Common@19.x` and there are no useful [changes between 18.x and 19.x](https://github.com/OctopusDeploy/Calamari/compare/18.1.4...octopusdeploy:19.1.0) in this context.